src/weather_model_graphs/save.py

```diff
@@ -106,11 +106,14 @@ def _concat_pyg_features(
     if list_from_attribute is not None:
         # create a list of graph objects by splitting the graph by the list_from_attribute
         try:
-            sub_graphs = list(
-                split_graph_by_edge_attribute(
-                    graph=graph, attr=list_from_attribute
-                ).values()
-            )
+            sub_graphs = [
+                kv[1]
+                for kv in sorted(
+                    split_graph_by_edge_attribute(
+                        graph=graph, attr=list_from_attribute
+                    ).items()
+                )
+            ]
```

Comment:
Minor change, but in my opinion better readability
```suggestion
            sub_graphs = [
                value
                for key, value in sorted(
                    split_graph_by_edge_attribute(
                        graph=graph, attr=list_from_attribute
                    ).items()
                )
            ]
```

Reply:
Agreed!